domain/domain.go

```diff
@@ -1642,6 +1645,14 @@ func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager)
 			if err != nil {
 				logutil.BgLogger().Debug("dump column stats usage failed", zap.Error(err))
 			}
+
+		case <-goTunerTicker.C:
+			if cgroup.InContainer() {
+				_, err := cgroup.SetGOMAXPROCS()
+				if err != nil {
+					logutil.BgLogger().Error("fail to sync GOMAXPROCS with instance", zap.Error(err))
```

Comment:
If in an env always return error, it will print error log every 3 seconds. Maybe use debug log is better.